lib/field.ts

```diff
@@ -65,6 +65,14 @@ export default class Field {
     assertHasLocale(this, locale)
     return this._fieldLocales[locale]
   }
+
+  getLocaleSpecificField(locale) {
```

Comment:
I'd name it `getForLocale(localeCode)`;

you'll be calling it like this: `sdk.fields[id].getForLocale(localeCode)` - field is already mentioned